x/time_test.go

```diff
@@ -0,0 +1,18 @@
+package x
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestRandomDelay(t *testing.T) {
+	base := time.Millisecond * 20
+	deviation := time.Millisecond * 5
+	start := time.Now()
+	RandomDelay(base, deviation)
+	delay := time.Now().Sub(start)
```

Comment:
```suggestion
	delay := time.Since(start)
```
`gosimple` will complain otherwise :wink:

Reply:
thanks, fixed!